app/src/main/AndroidManifest.xml

```diff
@@ -101,9 +101,10 @@
             android:value=".ui.SearchActivity" />
 
         <!-- Activities -->
-        <activity android:name=".ui.ConversationList"
+        <activity android:name=".ui.ConversationActivity"
             android:launchMode="singleTop"
-            android:theme="@style/Theme.Light">
+            android:theme="@style/Theme.Light"
+            android:configChanges="orientation|screenSize">
```

Comment:
ok thx, I didn't know how to do that. In the onCreate() call caused by the screen rotation the sliding pane was set to be closed even though it was visibly open.